File: ACL/backend/rest-api/model/file.go
package model

type File struct {
	Id           int64  `json:"file_id,string,omitempty" key:"primary" column:"file_id"`
	FileName     string `json:"file_name" column:"file_name"`
	FilePath     string `json:"file_path" column:"file_path"`

}

func (file *File) Table() string {
	return "files"
}


func (file *File) String() string {
	return Stringify(file)
}

File: ACL/frontend/ACLProject/src/app/file.ts
export class File {
    public file_id: number;
    public file_name: string;
    public file_path: string;
    constructor(file_id: number,file_name: string,file_path:string)
    {
    this.file_id=file_id;
    this.file_name=file_name;
    this.file_path=file_path;
    }
 }